contracts/optimistic-ethereum/libraries/trie/Lib_MerkleTrie.sol

```diff
@@ -145,9 +145,7 @@ library Lib_MerkleTrie {
     {
         // Special case when inserting the very first node.
         if (_root == KECCAK256_RLP_NULL_BYTES) {
-            return keccak256(
-                _makeLeafNode(_key, _value).encoded
-            );
+            return getSingleNodeRootHash(_key, _value);
```

Comment:
Refactor to use `getSingleNodeRootHash` since the logic was the same in both places.